back-end/src/base/materials/materials.module.ts

```diff
@@ -7,9 +7,17 @@ import { UsersRepository } from 'src/base/users/users.repository';
 import { MaterialsRepository } from 'src/base/materials/materials.repository';
 import { MaterialsService } from 'src/base/materials/materials.service';
 import { MaterialsController } from 'src/base/materials/materials.controller';
+import { JwtModule } from '@nestjs/jwt';
+import { config } from '@/config/config';
 
 @Module({
-	imports: [DatabaseModule, forwardRef(() => AuthModule)],
+	imports: [
+		DatabaseModule,
+		JwtModule.register({
+			secret: config.jwt.secret,
+			signOptions: { expiresIn: '1d' },
```

Comment:
Just to be sure, do you need a jwt for the materials ?
I don't see any jwt being created in the service.